homeassistant/components/sonos/media_player.py

```diff
@@ -255,6 +257,23 @@ def __init__(self, speaker: SonosSpeaker) -> None:
         self._attr_unique_id = self.soco.uid
         self._attr_name = self.speaker.zone_name
 
+    async def async_added_to_hass(self) -> None:
+        """Handle common setup when added to hass."""
+        await super().async_added_to_hass()
+        self.async_on_remove(
+            async_dispatcher_connect(
+                self.hass,
+                f"{SONOS_MEDIA_UPDATED}",
```

Comment:
No need for string interpolation.
```suggestion
                SONOS_MEDIA_UPDATED,
```

Isn't this a very generic signal? DOes this mean that all Sonos speakers update whenever this fires?

Reply:
The signal carries a speaker UID as the payload. Each `media_player` checks to see if its own UID or its coordinator's matches.